cluster-scope/base/core/namespaces/physrisk-resilience/kustomization.yaml

```diff
@@ -1,9 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- namespace.yaml
+  - namespace.yaml
 namespace: physrisk-resilience
 components:
-- ../../../../components/limitranges/default
-- ../../../../components/resourcequotas/small
-- ../../../../components/project-admin-rolebindings/physrisk
+  - ../../../../components/limitranges/default
```

Comment:
I also think we should remove limitrange as memory/cpu for individual containers for this app probably will be higher

Reply:
Our `limitranges/default` component assigns defaults only. It gets applied/assigned only if there are no `resources.limits` specified in the pod. If you assign those values in pod spec, nothing changes for you. If you have quota on a namespace you **are required** to have limitranges defaults, otherwise the pod without resource restriction fails to schedule. Removing this will cause issues, please don't do that.

https://github.com/operate-first/apps/blob/e446586e1a60c6dea17fd7b0baca85fc821d9819/cluster-scope/components/limitranges/default/limitrange.yaml#L6-L13

Reply:
@tumido Roger that.